### DISCUSSION TITLE
Copilot.

### DISCUSSION THREAD

**Discussion by thepeppapower** [MaaAssistantArknights/MaaAssistantArknights#10248] [Q&A] [answered]
Newbie to MAA. The question is, is it possible to make your own file for copilot, for example for TN-1. If so, how to? Thanks in advance.

**Comment by ABA2396** [comment]
prts.plus

But the interface is only in Chinese.

**Reply by thepeppapower** [reply]
I mean, I can personally make my own version of squad which will work on this site?

**Reply by ABA2396** [reply, marked as answer]
I'm not sure what you mean. Do you mean write your own Copilot guide?
![image](https://github.com/user-attachments/assets/e71b13e5-a005-4532-a034-2c61824fe4ed)
![image](https://github.com/user-attachments/assets/14be081a-b7a0-47d6-884d-d5676cdb6d93)
![image](https://github.com/user-attachments/assets/c52b116d-7acd-4385-9f73-0dac20702c72)
![image](https://github.com/user-attachments/assets/b32bde40-b03a-41f7-93ef-7db65546bdfe)

Or refer to this to write json directly
https://maa.plus/docs/en-us/protocol/copilot-schema.html

**Reply by thepeppapower** [reply]
Oh, thanks. It's perfect
